tls/s2n_tls_parameters.h

```diff
@@ -62,7 +62,7 @@
 #define TLS_ECDHE_BIKE_RSA_WITH_AES_256_GCM_SHA384 0xFF, 0x04
 #define TLS_ECDHE_SIKE_RSA_WITH_AES_256_GCM_SHA384 0xFF, 0x08
 #define TLS_EXTENSION_PQ_KEM_PARAMETERS 0xFE01
-#define TLS_PQ_KEM_EXTENSION_ID_BIKE1_R1_LEVEL_1 1
+#define TLS_PQ_KEM_EXTENSION_ID_BIKE1_L1_R1 1
```

Comment:
Overall, the changes look good. I'm not sure if we should push this before the spec is updated (perhaps it doesn't matter).

Reply:
We have consensus on the name changes. The spec will be updated; in the meantime, we can merge this.